packages/mask/src/plugins/IdeaMarket/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const SUBGRAPH_URL = 'https://subgraph.backend.ideamarket.io/subgraphs/name/Ideamarket/Ideamarket/'
+export const TESTNET_SUBGRAPH_URL =
+    'https://subgraph-test-avm-l2.backend.ideamarket.io/subgraphs/name/Ideamarket/IdeamarketTESTAVML2/'
+
+export const BASE_URL = 'https://ideamarket.io'
+export const TWITTER_BASE_URL = 'https://twitter.com'
+export const ToS = 'https://docs.ideamarket.io/legal/terms-of-service'
+
+export const ideaMarketHostnames = ['ideamarket.io']
+export const ideaMarketPathnameRegexMatcher = /^\/i\/(\S+)\/(\S+)/
+
+export const TWITTER_BEARER_TOKEN =
```

Comment:
do you sure this token can be published?

Reply:
Therefore I encapsulated api token, see commit: 696067d8c0f9be7d090c5c82c983043f98c3eb34